docker/Dockerfile.ci_i386

```diff
@@ -20,7 +20,9 @@
 
 FROM i386/ubuntu:18.04
 
-RUN apt-get update --fix-missing && apt-get install -y ca-certificates
+RUN apt-get update --fix-missing \
+    && apt-get install -y ca-certificates \
+    && rm -rf /var/cache/apt/archives /var/lib/apt/lists/*
```

Comment:
could you move this incantation into a script e.g. `/usr/local/bin/apt-install-and-clear` or something? and add a tests/lint/ script to verify that "apt-get install" and "apt install" don't appear anywhere inside docker/?